ui/src/app/store/machine/selectors.test.ts

```diff
@@ -0,0 +1,195 @@
+import {
```

Comment:
These are not new files as the diff suggests. They have the same changes as the other selector tests.